cmd/demoserver/kvdatamake.go

```diff
@@ -119,15 +142,26 @@ func generateData(folderName string, recorders int) int {
 	return fileNum
 }
 
+func DataService(ctx context.Context) {
+	grpcServer := grpc.NewServer()
+	pb.RegisterDataRWServiceServer(grpcServer, NewDataRWServer(ctx))
+	lis, err := net.Listen("tcp", PORT)
+	if err != nil {
+		fmt.Println("error happened ")
```

Comment:
Can use `log.Panic` to terminate process when error happens.

Reply:
fix it ,replace it with the log system